stable/influxdb/README.md

```diff
@@ -70,3 +70,13 @@ $ helm install --name my-release -f values.yaml stable/influxdb
 The [InfluxDB](https://hub.docker.com/_/influxdb/) image stores data in the `/var/lib/influxdb` directory in the container.
 
 The chart mounts a [Persistent Volume](kubernetes.io/docs/user-guide/persistent-volumes/) volume at this location. The volume is created using dynamic volume provisioning.
+
+## Starting with authentication
+
+In `values.yaml` change `.Values.config.http.auth_enabled` to `true`.
+
+Influxdb requires also a user to be set in order for authentication to be enforced. See more details [here](https://docs.influxdata.com/influxdb/v1.2/query_language/authentication_and_authorization/#set-up-authentication).
+
+To handle this setup on startup, a job can be enabled in `values.yaml` by setting `.Values.setDefaultUser.enabled` to `true`.
+
+Make sure to uncomment or configure the job settings after enabling it.
```

Comment:
Adding a note that if you don't set a password, a 'random' one will be created would be nice here.